apps/courses/src/routes/terms-of-service/index.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from 'react';
```

Comment:
As per my previous comments, this entire file should be deleted, along with the folder.